Diagnostic/diagnostic.py

```diff
@@ -728,8 +728,6 @@ def report_new_mdsd_errors(err_file_path, last_error_time):
     if len(last_error) > 0 and (datetime.datetime.now() - last_error_time) < datetime.timedelta(minutes=30):
         # Only recent error logs (within 30 minutes) are reported.
         hutil.log("Error in MDSD:" + last_error)
-        hutil.do_status_report(g_ext_op_type, "success", '1',
-                               "message in mdsd.err:" + str(last_error_time) + ":" + last_error)
```

Comment:
Could we instead do a more targeted fix that logs 409 errors from Azure Storage in this scenario as info/warning instead?